Makefile

```diff
@@ -113,11 +113,11 @@ $(GOIMPORTS):
 
 # install vendored dependencies
 vendor: Gopkg.lock
-	$(DEP) ensure -v --vendor-only
+	$(DEP) ensure --vendor-only
```

Comment:
I'm telling @bacongobbler on you

Reply:
:scream:

Reply:
I couldn't take the verbosity part of this, it was a bit too...verbose 😏